style/client.css

```diff
@@ -2906,6 +2906,31 @@ a.ilink.yours {
 	width: 95px;
 	text-align: right;
 }
+.changeform {
+	cursor: pointer;
+}
+.changeform i{
+	cursor: pointer;
+	display: block;
+	margin: -12px 0 -6px 0;
+	padding-left: 4px;
+	padding-top: 2px;
+	width: 12px;
+	height: 14px;
+	position: relative;
+	left: 90px;
+	border: 1px solid #569;
+	border-radius: 2px;
+	color: #569;
+	-webkit-box-shadow: 0 1px 2px rgba(0,0,0,.2);
+	-moz-box-shadow: 0 1px 2px rgba(0,0,0,.2);
```

Comment:
Do we still need these? @Zarel

I see we are seldom using the prefixed versions for ``box-shadow``